README.md

```diff
@@ -25,7 +25,7 @@ First, see the help:
 ```sh
 $ ./a11ym --help
 
-  Usage: a11ym [options] <url>
+  Usage: a11ym [options] url …
```

Comment:
I would write `a11ym [options] url [url...]` seems more clear to me, but like you prefer :)